include/ignition/msgs/CMakeLists.txt

```diff
@@ -1,3 +1,6 @@
 ign_install_all_headers(
   GENERATED_HEADERS
-    MessageTypes.hh)
+    MessageTypes.hh
+  INSTALL_BUT_DONT_INCLUDE
```

Comment:
I don't think that's a real parameter

* https://github.com/ignitionrobotics/ign-cmake/blob/ign-cmake2/cmake/IgnUtils.cmake#L601-L605

Reply:
I think you want to use `EXCLUDE_FILES` and then a separate install line

https://github.com/ignitionrobotics/sdformat/blob/sdf12/include/sdf/CMakeLists.txt#L1-L8

Reply:
oh, I missed https://github.com/ignitionrobotics/ign-cmake/pull/234

Reply:
Updated to your suggestion in [1c03d9e](https://github.com/ignitionrobotics/ign-msgs/pull/245/commits/1c03d9e91b26b924d1f178d5517e3938d7ec7c31)